virttest/utils_misc.py

```diff
@@ -1732,6 +1733,23 @@ def read_from_node_meminfo(self, node_id, key):
         """
         return self.get_all_node_meminfo()[node_id][key]
 
+    def get_online_nodes_withmem(self):
```

Comment:
I see another PR working on this same API https://github.com/avocado-framework/avocado-vt/pull/1134